R/module_nested_tabs.R

```diff
@@ -247,9 +292,9 @@ srv_teal_module.teal_module <- function(id,
         # wait for module_teal_data() to be not NULL but only once:
         ignoreNULL = TRUE,
         once = TRUE,
-        eventExpr = module_teal_data_validated(),
+        eventExpr = trigger_module(),
         handlerExpr = {
-          module_out(.call_teal_module(modules, datasets, module_teal_data_validated, reporter))
+          module_out(.call_teal_module(modules, datasets, module_teal_data, reporter))
```

Comment:
Should `trigger_data` and later calls be dependent on `module_teal_data_validated` or `module_teal_data`?